Test-NetStack.psm1

```diff
@@ -908,8 +910,19 @@ Function Test-NetStack {
                         Write-Host ":: $([System.DateTime]::Now) :: [Started] N -> Interface $($thisSource.InterfaceIndex) ($($thisSource.IPAddress))"
                         ":: $([System.DateTime]::Now) :: [Started] N -> Interface $($thisTarget.InterfaceIndex) ($($thisSource.IPAddress))" | Out-File $LogFile -Append -Encoding utf8 -Width 2000
 
+                        $StartTime = Get-Date
                         $thisSourceResult = Invoke-NDKPerfNto1 -Server $thisSource -ClientNetwork $ClientNodes -ExpectedTPUT $Definitions.NDKPerf.TPUT
-                       
+                        $EndTime = Get-Date
+
+                        $events = (Get-EventLog System -InstanceId 0x466,0x467,0x469,0x46a -ComputerName $thisSource.NodeName)
+
+                        if($events) { 
```

Comment:
The stage should error. This isn't just a log file, it should be in the NetStackResults object correct?